config/config.go

```diff
@@ -451,6 +453,8 @@ type TiKVClient struct {
 	// If a store has been up to the limit, it will return error for successive request to
 	// prevent the store occupying too much token in dispatching level.
 	StoreLimit int64 `toml:"store-limit" json:"store-limit"`
+	// StoreLivenessTimeout in nanosecond is the timeout for store liveness check request.
```

Comment:
The default value is in ms, but this comment is in ns.
How about making this option a duration string for easy to use?

Reply:
done~